auditlog/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# © 2015 ABF OSIELL <https://osiell.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': "Audit Log",
+    'version': "12.0.1.0.0",
+    'author': "ABF OSIELL,Odoo Community Association (OCA)",
+    'license': "AGPL-3",
+    'website': "https://www.osiell.com",
+    'category': "Tools",
+    'depends': [
+        'base',
+    ],
+    'data': [
+        'security/ir.model.access.csv',
+        'data/ir_cron.xml',
+        'views/auditlog_view.xml',
+        'views/http_session_view.xml',
+        'views/http_request_view.xml',
+    ],
```

Comment:
add image key as cover to improve the visibility in the apps store

Reply:
have to leave it to next PR, I am not good at icon design :)